share/governance.md

```diff
@@ -0,0 +1,80 @@
+
+VoxPupuli Governance
+====================
+
+
+Roles and Responsibilities
+--------------------------
+
+In order to have a smoothly running project, formal roles with corresponding responsibilities are established. A member of the community may have multiple roles.
+
+
+Users
+-----
+
+
+Users are community members who have a need for the project. They are the most important members of the community: without them, the project would have no purpose. Anyone can be a user; there are no specific requirements. Users should be encouraged to participate in the life of the project and the community as much as possible. User contributions enable the project team to ensure that they are satisfying the needs of those users. Common user activities include (but are not limited to):
+
+* advocating for use of the project
+* informing developers of project strengths and weaknesses from a new user’s perspective
+* providing moral support (a ‘thank you’ goes a long way)
+* writing documentation and tutorials
+* filing bug reports and feature requests
+* participating on the discussion board
+
+
+Users who continue to engage with the project and its community will often find themselves becoming more and more involved. Such users may then go on to become contributors, as described below.
+
+*How to become one*: Use a VoxPupuli project
+
+
+Collaborators
+-------------
+
+
+Collaborators are community members who submit patches to the project. These patches may be a one-time occurrence or occur over time. Expectations are that contributors will submit patches that are small at first and will only grow larger once the contributor has built confidence in the quality of their patches.
```

Comment:
The entire second sentence isn't sitting well, most likely because of the "Expectations are…" This implies some sort of criteria or suitability of a sort. There are Capital-E Expectations of collaborators. Perhaps instead something like, "It is anticipated that collaborators' contributions may be small at first, growing more considerable once the collaborator has built up the necessary knowledge, experience, and confidence."

Also: inconsistent use of "collaborator" and "contributor" in this paragraph, which can lead to confusion with the next section.